.github/workflows/trigger-load-tests.yml

```diff
@@ -0,0 +1,48 @@
+name: Trigger Load Tests
+
+on:
+    workflow_dispatch:
+        inputs:
+            tests:
+                description: >
+                    List of test names. This needs to be filled only if you want to run a specific set of tests. Example: foo,bar
+                required: false
+
```

Comment:
Better to add a cluster name input also. when we want to run the load test before another test finishes, we need to change the cluster name because the test framework sometimes fails to create the clusters when another cluster is there with the same name

Reply:
Fixed with ba7609d608087066771242721fd2a3fa956b546e